src/Compilers/Core/CommandLine/BuildProtocol.cs

```diff
@@ -397,8 +400,12 @@ public static CompletedBuildResponse Create(BinaryReader reader)
             var utf8Output = reader.ReadBoolean();
             var output = ReadLengthPrefixedString(reader);
             var errorOutput = ReadLengthPrefixedString(reader);
+            if (!string.IsNullOrEmpty(errorOutput))
+            {
+                throw new InvalidOperationException();
```

Comment:
Could we simply ignore the errorOutput? In any case, consider adding a comment about why it is wrong to have it.